docs/modules/ROOT/pages/data-migration.adoc

```diff
@@ -1,39 +1,64 @@
+= Transfer Data to Another Cluster
+:description: In this tutorial, you'll transfer data from one cluster to another using xref:wan-replication.adoc[WAN Replication]. You'll also learn how to safely delete a cluster after the successful transfer of data.
+
 
 {description}
 
-== Before you Begin
+== Context
+
+WAN replication supports the following use cases:
 
-You cannot use WAN Replication if your cluster has developer mode turned on.
+* Migrate data to a new cluster before decommissioning your original cluster.
 
-You need at least two clusters with the same configuration. For example, if one cluster
-contains xref:custom-classes-upload.adoc[custom classes] or Persistence,
-you should enable and configure these options for the other cluster as well.
+* Use WAN replication with a xref:failover-clients-with-hazelcast-cloud.adoc[failover client] as part of your disaster recovery strategy. WAN replication automatically synchronizes data between clusters. If a cluster fails, the failover client can switch to an alternative cluster with minimal interruption to service.
+
+== Before you Begin
 
-== Step 1. Create WAN Replication
+* Set up two active clusters with developer mode turned off. Use the same configuration for each cluster. For example, if one cluster
+contains xref:custom-classes-upload.adoc[custom classes] or persistence,
+enable and configure the same options for the other cluster.
 
-In order to create WAN Replication, you can go to *Cluster Detail* > *Settings* > *Wan Replication* and click *Create New Replication*
+* Decide which cluster you want to:
+** Send or migrate data from. This is the _publisher_ cluster.
+** Receive data. This is the _consumer_ cluster.
 
-After setting all these parameters, you can click *Next* to select and configure consumer cluster.
+== Step 1. Set Up WAN Replication
 
-Settings on this screen for consumer cluster are same as publisher cluster except *Active/Passive* section. If you set consumer cluster as *Active*, data will be synchronized from/to publisher and consumer. If you set consumer cluster as *Passive*, data will be sent only from publisher to consumer.
-Once you configured settings for consumer, you can click *Save*
+Set up WAN replication on the publisher cluster to send data to your consumer cluster.
 
-Within seconds, wan replication will be ready there.
+. Go to *Your Clusters* and select your publisher cluster.
+. Select *MANAGE CLUSTER > SETTINGS*.
+. On the *Cluster Settings* page, select *WAN Replication* and then click *+ CREATE NEW REPLICATION*.
++
+The publisher cluster is set as the *Active* cluster. Any data updates performed on this cluster are automatically copied to the other cluster connected by the WAN replication. In this case, your consumer cluster.
++
+. Update the xref:wan-replication.adoc#creating-wan-replication[other parameters] as required.
```

Comment:
Will you provide values for the user to input and explain them rather than linking? Tutorials should be self-contained. We shouldn't make the user go somewhere else and get lost.

Reply:
I've decided to leave the default values and provide a link in 'Learn more'/related links at the bottom of the page. Is there anyone I can check with to see what settings are recommended for the two use cases.

Reply:
Full description of parameters and values has now been added.